programs/vote_api/src/vote_instruction.rs

```diff
@@ -16,6 +16,7 @@ use solana_sdk::{
     instruction_processor_utils::DecodeError,
     pubkey::Pubkey,
     system_instruction, sysvar,
+    sysvar::rent,
```

Comment:
nit: would prefer you left sysvar:: on rent calls

irrelevant nit: prefer sysvar::{self, rent}  to sysvar twice

Reply:
> nit: would prefer you left sysvar:: on rent calls
> 
> irrelevant nit: prefer sysvar::{self, rent} to sysvar twice

done